Open Judge System/Data/OJS.Data/Migrations/202306071340433_ExectuionCommentAndExceptionType.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace OJS.Data.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+    
+    public partial class ExectuionCommentAndExceptionType : DbMigration
+    {
+        public override void Up()
+        {
+            AddColumn("dbo.Submissions", "ExecutionComment", c => c.String());
```

Comment:
We are using automatic migrations in the old Judge, so no need to add manual migration. Do test it first though, as I haven't made automatic migrations recently..